Responsive Mansion

Kiran-512 opened the pull request:
Hey there, Please review the code and advise if any corrections,

Kiran-512 committed:
commit

Kiran-512 committed:
project added

Kiran-512 committed:
Merge branch 'master' of https://github.com/NextGenDevOrg/02-CSS-Learning into Kiran-CSS-file

Kiran-512 committed:
readme added

Kiran-512 committed:
files renamed and link added for typerwriter

Kiran-512 committed:
flex added to the box

Rehan-Khan-Engineer reviewed (commented):
In the masonary layout, contents inside first grid area/cell is not centered. One way to align center every grid content is to give it flex. This way you don't have to give margin to center it. see the following link for example 
https://rehan-khan-engineer.github.io/Responsive-Masonary-Layout-CSS-Grid/

Kiran-512 committed:
alt value added

Kiran-512 commented:
So sorry, I had added a display flex to the grid items but forget to add the same in my personal repo so the link showed the old version in which I had initially added margin. It's updated in the code and also link too now, added alt value in recent commit updated pls review now.

Kiran-512 committed:
new project added

Kiran-512 committed:
Merge branch 'master' of https://github.com/NextGenDevOrg/02-CSS-Learning into Kiran-CSS-file

Kiran-512 committed:
footer width issue resolved

Kiran-512 committed:
e-commerce project finalised
